src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/tasks-top-extra.jelly

```diff
@@ -30,10 +30,10 @@ THE SOFTWARE.
     <j:if test="${computation != null}">
         <j:set var="title" value="${computation.displayName}"/>
     </j:if>
-    <l:task icon="images/24x24/notepad.png" href="${url}/computation/" title="${title}">
+    <l:task icon="images/24x24/notepad.png" href="${computation.getUrl()}" title="${title}">
```

Comment:
That seems to be true, so change `computation` and `getComputation` to be `@Nonnull` (with a note that they may be null temporarily inside the constructor, so subclasses beware), simplify `getCauseOfBlockage()` and `getEstimatedDuration()` accordingly, and simplify this view so it does no null check and inlines `${computation.displayName}`.